### DISCUSSION TITLE
Weather on terminal watch face

### DISCUSSION THREAD

**Discussion by AlbeyAmakiir** [InfiniTimeOrg/InfiniTime#1999] [Ideas]
I like the terminal watch face, and I like to know the current weather. I think it would be nice to have the latter on the former, especially since another watch face has implemented it already and can be used as a model.

The other entries on the watch face have so far avoided using icons like nerd fonts, so I imagine it would be in the format of `T°U Weather` where `T` is the temperature, `U` is the unit, C or F, and `Weather` is a one word description, such as Overcast or Sunny.

**Comment by JustScott** [comment]
@AlbeyAmakiir I added weather to it in this PR: #2001
